ios/Classes/ARKitManager.swift

```diff
@@ -7,7 +7,6 @@ class ARKitManager: NSObject, ARSessionDelegate, ARSCNViewDelegate {
     
     var trackingStateOK: Bool = false
     var lastFrame: ARFrame!
```

Comment:
Are these variables needed? At least they are not used in your code but maybe they are required by the implementation?